apps/core/tests/test_utils/dummy_instance.py

```
@@ -0,0 +1,7 @@
+from apps.core.tests.test_utils.dummy_meta import DummyMeta
+
+
+class DummyInstance:
+    def __init__(self, app_label: str, model_name: str, upload_folder_name: str | None):
```

Ruff (0.15.5)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)